news/487.bugfix.md

```diff
@@ -1 +1 @@
-Fix the format of dependency arrays when a new valud is appended.
+Fix the format of dependency arrays when a new value is appended.
```

Comment:
Given that https://github.com/pdm-project/pdm/issues/487 was an upstream issue with atoml, it should be removed/get rephrased maybe?

(Note that the dep was not bumped to enforce the fix)

Reply:
This commit alone also fixes the issue

Reply:
Ok, let's fix the typo then only?  (i.e. merge this)